### ISSUE TITLE
ws-qvh not working on Ubuntu

### ISSUE THREAD

**Issue by delphi007** [NetrisTV/ws-qvh#17]
ws-qvh is very cool and is working very well on Mac. I want it to be run on Ubuntu but fail. There is no video data output. I have tried to update to the latest [quicktime_video_hack](https://github.com/danielpaulus/quicktime_video_hack) project. The output is the same.

Following is what I have done.

1. upgrade to latest quicktime_video_hack with 2 patch from ws-qvh
2. Modify main.go to fit with latest go-ios which must change go-ios/usbmux to go-ios/ios
3. build ws-qvh-new on ubuntu
[on linux] 
4. run sudo usbmuxd &
5. run idevicepair pair
6. run ws-qvh-new 0.0.0.0:8888
7. run modified simple web client with h246 renderer connect to ws://<ip>:8888
8. No video output. Find the console output for ws-qvh-new from below 
9. Original ws-qvh is with same result.
10. I can stream video with ws-qvh & ws-qvh-new on mac.

![image](https://user-images.githubusercontent.com/11587194/153790255-7b32efc7-3dc7-4de9-9e9d-34e6b3477775.png)
...
![image](https://user-images.githubusercontent.com/11587194/153790380-c1795b72-b948-4dc3-b869-13f1c2b0ee17.png)

**Comment by drauggres** [comment]
Hi.
I never tried to run ws-qvh on Ubuntu, because I also need to run WebDriverAgent and it was possible only on macos.
What about quicktime_video_hack on ubuntu, does it work? I guess, you know golang and noticed that ws-qvh doesn't do much except video stream proxing. All real work is done by quicktime_video_hack.

**Comment by delphi007** [comment]
@drauggres Yes, exactly! I have not successfully run quicktime_video_hack on either Linux or Mac. But can successfully run  ws-qvh on Mac. So I try ws-qvh on linux to see if I could be luck enough. :) 
So the next step I could take is to study why quicktime_video_hack not running well on mac. 

Thank you, Drauggres!

**Comment by LWHikarik** [comment]
Hi, the current issue is that fundamental conflict: to switch device into QT mode you need to enable usbmux-driver. But when driver enabled - qvh cannot recapture the image and we getting stuck in infinite chicken-egg loop. Another problem is that Linux is not receiving ping from latest IOS, and H264 bytes are not coming. I will try to do go-ios migration, in theory it will work on Linux also
